specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2020-03-01-preview/examples/CheckNameAvailability_AlreadyExists.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "api-version": "2020-03-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "locationName": "westus2",
```

Comment:
this property is not in the swagger - remove

Reply:
Removed from both